README.md

```diff
@@ -83,6 +83,8 @@ download the most recent GitHub Enterprise version.
     host name. Additional options are available and documented in the
     configuration file but none are required for basic backup functionality.
 
+    * In a clustering environment, the `GHE_EXTRA_SSH_OPTS` key must be configured.
```

Comment:
`key must be configured.` -> ```must contain the `-i <private key file>` ssh option.```

might read a bit better and is more explicit about the requirements